doc/build-windows-msvc.md

```diff
@@ -0,0 +1,82 @@
+# Windows / MSVC Build Guide
+
+This guide describes how to build bitcoind, command-line utilities, and GUI on Windows using Micsrosoft Visual Studio.
+
+For cross-compiling options, please see [`build-windows.md`](./build-windows.md).
+
+## Preparation
+
+### 1. Visual Studio
+
+This guide relies on using CMake and vcpkg package manager provided with the Visual Studio installation.
+Here are requirements for the Visual Studio installation:
+1. Minimum required version: Visual Studio 2022 version 17.6.
+2. Installed components:
+- The "Desktop development with C++" workload.
+
+The commands in this guide should be executed in "Developer PowerShell for VS 2022" or "Developer Command Prompt for VS 2022".
+The former is assumed hereinafter.
+
+### 2. Git
+
+Download and install [Git for Windows](https://git-scm.com/download/win). Once installed, Git is available from PowerShell or the Command Prompt.
+
+### 3. Clone Bitcoin Repository
+
+Clone the Bitcoin Core repository to a directory. All build scripts and commands will run from this directory.
+```
+git clone https://github.com/bitcoin/bitcoin.git
+```
+
+
+## Triplets and Presets
+
+The Bitcoin Core project supports the following vcpkg triplets:
+- `x64-windows` (both CRT and library linkage is dynamic)
+- `x64-windows-static` (both CRT and library linkage is static)
+
+To facilitate build process, the Bitcoin Core project provides presets, which are used in this guide.
+
+Available presets can be listed as follows:
+```
+cmake --list-presets
+```
+
+## Building
+
+CMake will put the resulting object files, libraries, and executables into a dedicated build directory.
+
+In following istructions, the "Debug" configuration can be specified instead of the "Release" one.
+
+### 4. Building with Dynamic Linking with GUI
+
+```
+cmake -B build --preset vs2022 -DBUILD_GUI=ON  # It might take a while if the vcpkg binary cache is unpopulated or invalidated.
+cmake --build build --config Release           # Use "-j N" for N parallel jobs.
+ctest --test-dir build --build-config Release  # Use "-j N" for N parallel tests. Some tests are disabled if Python 3 is not available.
+```
+
+### 5. Building with Static Linking without GUI
+
+```
+cmake -B build --preset vs2022-static          # It might take a while if the vcpkg binary cache is unpopulated or invalidated.
+cmake --build build --config Release           # Use "-j N" for N parallel jobs.
+ctest --test-dir build --build-config Release  # Use "-j N" for N parallel tests. Some tests are disabled if Python 3 is not available.
+cmake --install build --config Release         # Optional.
```

Comment:
It would be useful to know what the install command will do.

Reply:
It installs the built project to the default install prefix, which is the value of the `CMAKE_INSTALL_PREFIX` variable.

Reply:
> I'm doing some test builds on Windows and it was a bit of a treasure hunt to find the binaries.

@sipsorcery: Rather than going on a treasure hunt, wouldn't you "install" the built product to a staging directory and then produce your distributable package from there? That way you'll pick up all the bits that the build system wants you to have and none that it doesn't.

Reply:
> @sipsorcery: Rather than going on a treasure hunt, wouldn't you "install" the built product to a staging directory and then produce your distributable package from there? That way you'll pick up all the bits that the build system wants you to have and none that it doesn't.

I think that would be a reasonable soln. I'm used to the way msvc places all the executables in a single directory. GIven the `cmake --install` command provides an option to achieve the same that would work. 

As mentioned in another comment I think the `cmake --install` explanation should be enhanced to assist those that use it infrequently, like me.